src/index.html

```diff
@@ -9,6 +9,72 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <img src="images/logo.png" alt="image">
+      </div>
+      <nav class="navigation">
+        <ul class="container">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="is-active nav_link"
+
+            >APPLE
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >SAMSUNG
+            </a>
```

Comment:
fix all links' text like that
```suggestion
            <a
              href="#"
              class="nav_link"
            >
              SAMSUNG
            </a>
```